docs/tutorial/file.rst

```diff
@@ -105,6 +105,13 @@ The channels have a few convenient methods, like `.plot()` which make it easy to
     plt.plot(f1x_timestamps, f1x_data)
 
 The `timestamps` attribute returns absolute values in nanoseconds.
+Note that `timestamps` cannot be converted to floating point without losing precision::
+
+    >>> t = f1x_timestamps[0]
+    >>> roundtrip_t = np.int64(np.double(t))
```

Comment:
It's better to use `float64` since it's not platform-dependent. And it's a better showcase for the example: 64-bit integer vs 64-bit float. You may also want to mention that the precision loss is caused by the limited size of the mantissa.
```suggestion
    >>> roundtrip_t = np.int64(np.float64(t))
```

Reply:
Done and added:

`The reason for this is that timestamps exceed the maximum integer value representable by the mantissa.`

Unless you can think of a more user-friendly way of describing that.